documentation/features/context-menu.mdx

```diff
@@ -0,0 +1,46 @@
+---
+sidebarTitle: Context Menu
+title: Context Menu
+description: Learn how to use the context menu.
+---
+
+Context-aware menus enable quick actions like creating variables or adding query
+parameters by selecting text or elements. The context menu offers three options:
+**Set as Variable**, **Add to Parameters**, and **Open Request in New Tab**.
+
+<Info>
+  The "Add to Parameters" option is available only when using query parameters.
+</Info>
+
+## Set as Variable
```

Comment:
Just paraphrasing this section to add more clarity while reading:

> You can use the **Set as Variable** option from the context menu to assign a selected value to an environment variable.
> 
> Hoppscotch supports various variable scopes, including global, environment, and request-specific variables. For more details on these scopes, [click here](/documentation/features/variables).

can you make a new commit for this?